colpali_engine/utils/torch_utils.py

```diff
@@ -1,14 +1,33 @@
+import gc
+
 import torch
 
 
-def get_torch_device() -> str:
+def get_torch_device(device: str = "auto") -> str:
```

Comment:
Not sure to understand for this comment, can you clarify pls?

Reply:
Where is this get_torch_device function called. Can the user specify from ColPali.from_pretrained what device he wants ? I guess if you just call this with model.device as an arg it's okay since you can start model from_pretrained(device_map=X)

Reply:
For the moment it's only called:
- In `CustomRetrievalEvaluator`
- In tests.

It's not used in the models.